### PR TITLE
Add check_installed function in updater.py

### PR DESCRIPTION
**Pull request by Wescoeur**
Tested on XCP-ng 8.2 & 8.3.
Documentation updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,6 +108,13 @@ Update the host.
 $ xe host-call-plugin host-uuid=<uuid> plugin=updater.py fn=update
 ```
 
+#### `is_installed`:
+Test if one or many packages are installed.
```
Comment (benjamreis) [inline README.md:112]:
```suggestion
Return an array of the installed packages given in arguments with an empty entry for not installed ones.
```

**Review by benjamreis — APPROVED** [review]
Minor doc nitpick, otherwise LGTM

File: README.md (hunk 1 above)
Comment (Wescoeur) [reply README.md:112]: Not an array. It's JSON Object. ;)
So maybe:
```
Return a JSON object of the installed packages given in arguments with an empty value string for not installed ones.
```

Comment (benjamreis) [reply README.md:112]: Fine by me, I phrased like the method above to keep consistency but your suggestion is ok too.

File: SOURCES/etc/xapi.d/plugins/updater.py [hunk 2]
```diff
@@ -181,6 +181,33 @@ def check_update(session, args):
...
 
+@error_wrapped
+def query_installed(session, args):
+    packages = args.get('packages')
+    if not packages:
+        return '{}'
+
+    import re
+    packages = re.sub(r'\s+', ' ', packages).replace(',', ' ').split(' ')
+    packages = filter(lambda package: package, packages)
```
Comment (stormi) [inline SOURCES/etc/xapi.d/plugins/updater.py:192]: What input is supported (spaces, commas...) should be specified in the readme and tested in the unit tests.

**Pull request merged**